app/controllers/application_controller.rb

```diff
@@ -1,10 +1,17 @@
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
   verify_authorized unless: :devise_controller?
-
   before_action :set_current_user
   around_action :switch_locale
 
+  KNOWN_ERRORS = [ActionPolicy::Unauthorized]
```

Comment:
Should we put all error logging logic in a concern to include on application controller? Just keeps it in once place, out of the way of domain logic that might need to live on application controller.

Reply:
I would rather not in this case. The code isn't going to get reused anywhere else. And the order of many of the actions in ApplicationController are dependent on each other. The controller is small now too.